### ISSUE TITLE
Swarmpit nginx config

### ISSUE THREAD

**Issue by mrspartak** [swarmpit/swarmpit#456]
I tried to search but didn't find any configs of how to place swarmpit behind a reverse proxy and close 888 port to be directly accessible via server_ip:888.

I installed Nginx as a global service. Added a new overlay network and attached it to Nginx and smarpit_app.
Here is my Nginx config. 
```
upstream backend {
    server swarmui_app:8080;
}

server {
    listen 443 ssl;
    charset UTF-8;
    
    ssl_certificate    /run/secrets/labodina_bundle.crt;
    ssl_certificate_key    /run/secrets/labodina.key;
    
    server_name swarm.labodina.com;

    location ~ ^/(.*)$ {
        proxy_pass http://backend/$1;
        
        proxy_http_version 1.1;

        proxy_read_timeout 120s;

        proxy_set_header Host $host;
        
        proxy_set_header Upgrade $http_upgrade;
        proxy_set_header Connection "upgrade";
        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
        proxy_set_header X-Real-IP $remote_addr;
        proxy_set_header X-NginX-Proxy true;
    }
}
```

So nginx sees upstream, but can't route static paths. I get website but some requests and assets can't be reached.
For example /events path
![image](https://user-images.githubusercontent.com/993910/72900116-5dd62680-3d38-11ea-9078-91548de7088f.png)
Can't vew logs and get this error
```
{:spec "(spec-tools.core/spec {:spec (clojure.spec.alpha/keys :req-un [:spec$293/since]), :type :map, :leaf? false})", :problems [{:path [], :pred "(clojure.core/fn [%] (clojure.core/contains? % :since))", :val {}, :via [], :in []}], :type "reitit.coercion/request-coercion", :coercion "spec", :value {}, :in ["request" "query-params"]}
```

**Comment by nohaapav** [comment]
@mrspartak Reverse proxy seems to be broken since 1.8. See

https://github.com/swarmpit/swarmpit/issues/413

I'll push fix hopefully this week

**Comment by nohaapav** [comment]
Fix in 1.8 & latest
